src/lib/output/themes/lib.tsx

```diff
@@ -83,10 +81,11 @@ export function classNames(names: Record<string, boolean | null | undefined>, ex
 export function hasTypeParameters(
     reflection: Reflection
 ): reflection is Reflection & { typeParameters: TypeParameterReflection[] } {
-    if (reflection instanceof DeclarationReflection || reflection instanceof SignatureReflection) {
-        return reflection.typeParameters != null && reflection.typeParameters.length > 0;
-    }
-    return false;
+    return (
```

Comment:
I think I prefer the original if I ever had to step into this function, given that I don't think that's ever happened... fine as is.

Reply:
Yeah, this one is very much up to personal preference.